### PR TITLE
Ns 269/update rest endpoints for packets

### PR DESCRIPTION
**Pull request by net-illia-stetsenko**
Added initial endpoints for pcap management which need to be updated later after all the necessary backend is implemented

### CODE REVIEW

File: net-gateway/src/endpoints/networks/add_network.rs
Comment (net-yehor-tretiakov) [inline net-gateway/src/endpoints/networks/add_network.rs:1]: Add TODO about adding thing to inserter

**Review by net-yehor-tretiakov — COMMENTED** [review]
Lftm. If youll not forget about inserter part))

**Review by net-yehor-tretiakov — APPROVED** [review]

File: net-gateway/src/endpoints/packets/core/packet.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct Packet {
+    pub id: String,
+    #[serde(rename = "frameTime")]
+    pub frame_time: u64,
```
Comment (net-illia-stetsenko) [inline net-gateway/src/endpoints/packets/core/packet.rs]: Btw why did I leave frame_time not option

File: net-gateway/src/endpoints/networks/add_network.rs
Comment (net-illia-stetsenko) [reply net-gateway/src/endpoints/networks/add_network.rs:1]: Do you mean update the api or net-inserter in general?

**Pull request merged**